_blog/2024-01-24-using-cronjobs-rust.mdx

```diff
@@ -0,0 +1,294 @@
+---
+title: "Using Cronjobs in Rust"
+description: This article talks about how you can write cron jobs as a web service on Shuttle using the apalis cron job framework.
+author: josh
+tags: [cronjob, rust, tutorial, guide]
+thumb: 
+cover: 
+date: '2024-01-23T14:30:00'
+---
+
+In this article we’re going to talk about how you can write your own cron jobs as a web service using Shuttle! 
+
+Cron jobs (or “scheduled tasks”) are useful for many things. They allow you to automatically do things like:
+
+- Automate data back-ups
+- Adding daily reminders (for example to customers who are signed up to a service you own but haven’t started using it yet)
+- Creating/writing reports
```

Comment:
Just something I've noticed from the previous article, is that we're not consistent with adding `.` at the end of the sentences when they're part of a list. Personally I prefer to do it, but the most important IMHO is to be consistent.